src/app/docs/text-editor/text-editor-docs.component.html

```diff
@@ -20,5 +20,120 @@
       >
       </sky-docs-code-example>
     </sky-docs-code-examples> -->
+
+    <sky-docs-design-guidelines>
+
+      <sky-docs-demo-page-section
+        heading="Usage"
+      >
+        <sky-docs-design-guideline
+          heading="Use when"
+          headingStyle="success"
+        >
+          <p>
+            Use a text editor when users need to create blocks of text that include formatting, hyperlinks or merge fields within the text.
+          </p>
+          <sky-docs-design-guideline-thumbnail
```

Comment:
Do we need two images here? One for default theme and one for modern theme? Adam's been including two images, so I'm assuming we need that here as well?

Reply:
updated